src/GraphQl/Resolver/Factory/CollectionResolverFactory.php

```diff
@@ -118,6 +118,7 @@ public function __invoke(string $resourceClass = null, string $rootClass = null,
             if ($collection instanceof PaginatorInterface && ($totalItems = $collection->getTotalItems()) > 0) {
                 $data['pageInfo']['endCursor'] = base64_encode((string) ($totalItems - 1));
                 $data['pageInfo']['hasNextPage'] = $collection->getCurrentPage() !== $collection->getLastPage() && (float) $collection->count() === $collection->getItemsPerPage();
+                $data['totalCount'] = $collection->getTotalItems();
```

Comment:
Can you add a test please?

Reply:
> I will add tests later today.

Reply:
Fixed in a new commit. I will also add some Behat scenarios later to make sure it's populated.

Reply:
There is a variable handling this result declared 3 lines before (`$totalItems`) instead of the fonction call

Reply:
@antograssiot Thanks! fixed.